pkgs/tools/graphics/icoutils/default.nix

```diff
@@ -8,11 +8,25 @@ stdenv.mkDerivation rec {
     sha256 = "0wdgyfb1clrn3maq84vi4vkwjydy72p5hzk6kb9nb3a19bbxk5d8";
   };
 
-  buildInputs = [ libpng ];
+  buildInputs = [ makeWrapper libpng ];
+  perlLibs = [ perlPackages.LWP ];
+  propagatedBuildInputs = [ libpng perl perlPackages.LWP ];
```

Comment:
Does it still work if you do it like this:

````
 buildInputs = [ makeWrapper libpng perl ];
 propagatedBuildInputs = [ perlPackages.LWP ];
````

AFAICT `perlLibs` isn't used by anything and having native dependencies in `propagatedBuildInputs` makes the package unnecessarily depend on the dev headers of perl/libpng.